docs/_sidebar.md

```diff
@@ -17,7 +17,8 @@
 - **Manage password**
   - [Reset your TechPass password](reset-password)
 - **TechPass support resources**
-  - [TechPass status](techpass-status)
+  - [TechPass status](techpass-status.md)
+  - [Raise a service request](raise-a-service-request.md)
   - [FAQs](/support/overview.md)
   - [Terms and policies](terms-and-policies)
```

Comment:
how come others don't have .md?

Reply:
habit, sorry! removed. i dont think it affects anything tho